site/source/tutorials/1030_routing/index.md

```diff
@@ -0,0 +1,183 @@
+
+{% aside 'Important!' %}
+When using a `defaultRoute` in the routing configuration, then the `router` will need to be started *after* the projector is appended.
+{% endaside %}
+
+Finally to initialize the routing, the `start` needs to be called on the `router` instance.
+
+Next, we will create `outlets` to control when our widgets are displayed.
+
+{% section %}
+
+## Routing outlets
+
+{% task 'Create outlets for the routed widgets.' %}
+
+{% aside 'Reminder' %}
+The path that is associated to an outlet name is defined by the routing configuration from the first section.
+{% endaside %}
+
+An `Outlet` is a higher order component that wraps a widget, and controls whether the widget is rendered based on if the navigated route associated to the outlets name. To keep out application organized, outlets are usually stored in a separate directory called `outlets`.
+
+{% instruction 'Add the following code to `WorkerFormOutlet.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/outlets/WorkerFormOutlet.ts' %}
+
+The fist argument for the `Outlet` function determines the widget to display when the configured route is selected. Consider an `outlet` configured for a `path` of `foo`, its wrapped widget will render for a selected route `foo` (described as an `index` match). It will also display for any route that the outlet's `path` partially matches too, for example, `foo/bar` or `foo/bar/baz`.
+
+Normally this is perfectly acceptable, however there are scenarios, where it is necessary to explicitly define a widget for an `index` match. To support this more advanced configuration, the first argument also accepts an object that can be used to specify these components explicitly.
+
+```ts
+const ExampleOutlet = Outlet({
+	index: MyIndexComponent,
+	component: MyComponent,
+	error: MyErrorComponent
+}, 'outlet-name');
+```
+
+We use this option for both the `BannerOutlet.ts` and `WorkerContainerOutlet.ts`, as we do not want the widgets to render for routes unless the are an exact match for the selected route.
+
+{% instruction 'Add the following code to `WorkerContainerOutlet.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/outlets/WorkerContainerOutlet.ts' %}
+
+{% instruction 'Add the follow code to `BannerOutlet.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/outlets/BannerOutlet.ts' %}
+
+Now, we can the swap out the widgets for the newly created `outlets`. Notice we haven't needed to changed any of wrapped widgets!
+
+{% instruction 'First, swap the widget imports for the outlet imports in `App.ts`.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' lines:8-10 %}
+
+{% instruction 'Then, swap the widget usages for their outlets.' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/App.ts' lines:50-55,59-61 %}
+
+Running the application now, should display the `Banner.ts` by default but also enable routing to the other widgets using the `/#directory` and `/#new-widget` routes.
```

Comment:
application now, should display the `Banner.ts` by default but also -> application now should display the `Banner.ts` by default, but also